test/integration/features/basic.feature

```diff
@@ -31,7 +31,7 @@ Feature: Basic test
         When executing "crc status" fails
         Then stderr should contain 
         """
-        Machine "crc" does not exist. Use "crc start" to add a new one.
+        Machine 'crc' does not exist, use 'crc start' to create it.
```

Comment:
this can not be conjoined sentences this way. Full-stop should break the error from the solution. Or other words are needed to join the sentences, like 'and therefore', but that sounds wordy and does not ad meaning.

Reply:
Fixed.

Reply:
oops, typo; add meaning